zhaquirks/tuya/TS011F_plug.py

```diff
@@ -0,0 +1,112 @@
+"""TS011F plug."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, GreenPowerProxy, Groups, Identify, OnOff, Ota, Scenes, Time
+from zigpy.zcl.clusters.homeautomation import ElectricalMeasurement
+from zigpy.zcl.clusters.smartenergy import Metering
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODEL,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+class TuyaZBMetering(CustomCluster, Metering):
+    """TuyaZBMetering cluster implementation. This implementation divides the KwH for tuya devices"""
+
+    cluster_id = Metering.cluster_id
```

Comment:
```suggestion
```

No need to redefine this

Reply:
This is OK

also removed line 32: cluster_id = ElectricalMeasurement.cluster_id